### ISSUE TITLE
[INSTALL] Provide easier means of setup for Ubuntu 22.04

### ISSUE THREAD

**Issue by vii33** [voxel51/fiftyone#2480]
### System information

-   **OS Platform and Distribution** (e.g., Linux Ubuntu 16.04):  Ubuntu 22.04.1 LTS via WSL2
-   **Python version** (`python --version`):  Python 3.10.6
-   **FiftyOne version** (`fiftyone --version`):  failes (pip install from 04.01.2023)
-   **FiftyOne installed from** (pip or source):  pip

### Commands to reproduce

```zsh-shell
mkv  # create virtual env
pip install fiftyone
python
>>> import fiftyone as fo
```

### Describe the problem

With Ubuntu 22.04, `libssl1.1` is missing. This leads to an import error: #1803 
There is currently no easy way to install FiftyOne. I invested over 10 hours now and give up.

----

**Things I tried:** 

#### Default Installation
- Fails, because libssl1.1 is not distributed with Ubuntu 22.04. Manual installation procedure according to #1803 fails because OpenSSL is also missing and I could not get this to work.
- The hint in the documenation to upgrade `wheel` didn't help.

#### Alternative Linux builds
Followed the [section about alternative builds](https://voxel51.com/docs/fiftyone/getting_started/troubleshooting.html#alternative-linux-builds), and executed `pip install fiftyone-db-ubuntu2004`. 
Same error.

#### Building a docker image
Finally went all the way to create a Docker image. Build was successfull and fiftyone can be used in the terminal. 
```
docker run -it \                                                                                              
-p 5151:5151 \
voxel51/fiftyone
```
But trying to start the FiftyOne UI on local host simple leads to a blank HTML page.
```
<!DOCTYPE html>
<html lang="en">
  <head>
    <link rel="icon" href="[/favicon.ico](view-source:http://localhost:5151/favicon.ico)" />
    <link rel="preload" href="[https://fonts.googleapis.com/css2?family=Palanquin&display=swap](view-source:https://fonts.googleapis.com/css2?family=Palanquin&display=swap)" as="font" type="font/woff" crossorigin>
    <meta charset="UTF-8" />
    <meta name="description" content="Explore, Analyze, Curate" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>FiftyOne</title>
    <script type="module" crossorigin src="[/assets/index.36ee64bc.js](view-source:http://localhost:5151/assets/index.36ee64bc.js)"></script>
    <link rel="stylesheet" href="[/assets/index.73154351.css](view-source:http://localhost:5151/assets/index.73154351.css)">
  </head>
  <body>
    <div id="root"></div>
    <div id="teams"></div>
    <div id="modal"></div>
    
  </body>
</html>
```

---

### My humble request

- Please update the main documentation on how to install FiftyOne with **newer Ubuntu Versions. (Or update these probably outdated packages)**
- Docker Containers where inveted to avoid such getting-started nightmares. It would be great if the team could **provide an official image on Docker Hub**. Many other projects do this as well. I would provide one myself, if the installation would work.

.

**Comment by kaixi-wang** [comment]
Can you try installing in an environment with python version 3.9

**Comment by benjaminpkane** [comment]
Hi @vii33. You should be able to get set up by configuring your own MongoDB connection. More details are in my comment [here](https://github.com/voxel51/fiftyone/issues/1803#issuecomment-1371164010)

**Comment by vii33** [comment]
> Hi @vii33. You should be able to get set up by configuring your own MongoDB connection. More details are in my comment [here](https://github.com/voxel51/fiftyone/issues/1803#issuecomment-1371164010)

I did, thank you. I still think that a default container build would be very benifcial, though. As an example the setup of CVAT was very easy to do (docker pull, mount volume).

**Comment by rusmux** [comment]
@benjaminpkane Is there no other easier way to do this? It would be nice to have everything working just after `pip install`

**Comment by brimoor** [comment]
Ubuntu 22.04 users should now be able to install FiftyOne as follows:

```py
pip install fiftyone
pip install fiftyone-db-ubuntu2204  # installs MongoDB 6
```

Note: manually installing `fiftyone-db-ubuntu2204` will be necessary until we update the MongoDB version [here](https://github.com/voxel51/fiftyone/blob/e62d0a21aa70d0a05749d7d5324557a9b07d832f/package/db/setup.py#L29) to 6.0.4 or greater, at which point the `fiftyone` package will ship with MongoDB 6.0.4 by default on all Linux systems.

**Comment by brimoor** [comment]
See also: #3321, #3236, #2961, #2900, #2737, #1988, #1803

**Comment by benjaminpkane** [comment]
Fixed in 0.22.3!
